resources/clj-kondo.exports/taoensso/encore/taoensso/encore_hooks.clj

```diff
@@ -64,19 +64,30 @@
              binding-vec
              body))))}))
 
-(defn defonce
-  [{:keys [node]}]
+(defn -def-impl
+  [{:keys [node]} core-macro-sym]
   ;; args = [sym doc-string? attr-map? init-expr]
   (let [[sym & args] (rest (:children node))
-        [doc-string args]    (if (and (hooks/string-node? (first args)) (next args)) [(hooks/sexpr (first args)) (next  args)] [nil        args])
-        [attr-map init-expr] (if (and (hooks/map-node?    (first args)) (next args)) [(hooks/sexpr (first args)) (fnext args)] [nil (first args)])
-
+        [doc-string args] (if (and (hooks/string-node? (first args)) (next args))
+                            [(hooks/sexpr (first args)) (next args)]
+                            [nil args])
+        [attr-map init-expr] (if (and (hooks/map-node? (first args)) (next args))
+                               [(hooks/sexpr (first args)) (fnext args)]
+                               [nil (first args)])
         attr-map (if doc-string (assoc attr-map :doc doc-string) attr-map)
         sym+meta (if attr-map (with-meta sym attr-map) sym)
-        rewritten
-        (hooks/list-node
-          [(hooks/token-node 'clojure.core/defonce)
-           sym+meta
-           init-expr])]
-
+        rewritten (hooks/list-node
+                    [(hooks/token-node core-macro-sym)
+                     sym+meta
+                     init-expr])]
+    #_(println "old node:" node)
+    #_(println "new node:" rewritten)
```

Comment:
Leaving these for ease of debugging (in accordance with the [Hooks Tips & Tricks](https://github.com/clj-kondo/clj-kondo/blob/master/doc/hooks.md#debugging)).